starters/solidstart-tanstackquery-tailwind-modules/README.md

```diff
@@ -71,11 +71,11 @@ git clone https://github.com/thisdot/starter.dev.git
 
 ## Commands
 
-- `npm run dev` - Runs the development server on localhost port 3000 with HMR
+- `npm run dev` - starts a development server backed by Vite on localhost port 3000 with HMR
```

Comment:
Miss the capitol letter for "start" in the command description

Reply:
the port number is wrong also.

Reply:
👍🏾